apis/index.ts

```diff
@@ -0,0 +1,23 @@
+const basePath = `https://pokeapi.co/api/v2/pokemon`;
+
+const fetchPokemonList = async (limit: number, offset: number) => {
+  const response = await fetch(`${basePath}?limit=${limit}&offset=${offset}`, {
+    method: "GET",
+    headers: {
+      "Content-Type": "application/json",
+    },
+  });
+  return await response?.json();
```

Comment:
Handle error case please - what if fetch fails how do we handle that?

Reply:
Done as stated